tests/runner/kube_testplatform.go

```diff
@@ -316,6 +324,10 @@ func (c *KubeTestPlatform) Restart(name string) error {
 		return err
 	}
 
+	if _, err := m.WaitUntilDeploymentState(m.IsDeploymentDone); err != nil {
```

Comment:
Isn't this a dupe of line 286 from the Scale call above?

Reply:
Yes. let me remove this.